content/docs/guide.md

```diff
@@ -99,54 +269,41 @@ template by running the `init` command:
 # See the documentation for more information:
 # https://packit.dev/docs/configuration/
 
-specfile_path: packit.dev.spec
+specfile_path: directory.spec
 
 # add or remove files that should be synced
 files_to_sync:
-    - packit.dev.spec
+    - directory.spec
     - .packit.yaml
 
 # name in upstream package repository/registry (e.g. in PyPI)
-upstream_package_name: packit.dev
+upstream_package_name: directory
 # downstream (Fedora) RPM package name
-downstream_package_name: packit.dev
+downstream_package_name: directory
```

Comment:
This is the second time we show the output of `packit init` on this page, but I can't decide which one to remove, so I think I'm fine with it.
So just heads up to make sure you are aware of it.

Reply:
Yes, I am on the same boat with this. I can't decide..;)

Reply:
Well at this point, folks should already have their packit.yamls prepared, so I'd drop this one.